action.yaml

```diff
@@ -80,6 +85,7 @@ runs:
       run: |
         ${{ github.action_path }}/configure_macports "${{ inputs.parameters }}"
     - name: 'Cache MacPorts Installation'
+      if: ${{ inputs.enable-cache == 'true' || ( inputs.enable-cache == true && inputs.enable-cache != 'false' ) }}
```

Comment:
That seems rather convoluted. What is the reason for this? Why not just `${{ inputs.enable-cache }}` since the value is a boolean?

Reply:
Because I think that boolean inputs are actually parsed as string, instead of real booleans.